grammars/less.cson

```diff
@@ -300,85 +327,183 @@
     'match': ':'
     'name': 'punctuation.separator.key-value.css'
   }
+  {
+    'match': '\\btrue\\b'
+    'name': 'constant.language.boolean.less'
+  }
+  {
+    'match': '\\bdefault\\b'
+    'name': 'support.function.default.less'
+  }
+  {
+    'match': '\\b(iscolor|isnumber|isstring|iskeyword|isurl)\\b'
+    'name': 'support.function.type-checking.less'
+  }
+  {
+    'match': '\\b(ispixel|ispercentage|isem|isunit)\\b'
+    'name': 'support.function.unit-checking.less'
+  }
+  {
+    'include': '#font_name'
+  }
+  {
+    'include': '#comma'
+  }
+  {
+    'include': '#color_name'
+  }
+  {
+    'include': '#less_builtin_function'
+  }
+  {
+    'include': '#css_builtin_function'
+  }
+  {
+    'include': '#gradient_builtin_function'
+  }
 ]
 'repository':
-  'property_list':
-    'begin': '\\{'
+  'variable_interpolation':
+    'match': '@{[a-zA-Z0-9_-]+}'
+    'name': 'variable.other.interpolation.less'
+  'string_double':
```

Comment:
I would merge the `string_double` and `string_single` patterns into a single one called `strings` since they're always included together.